ci/platformio.ini

```
@@ -118,6 +118,14 @@ lib_ldf_mode = chain+
 board_microros_transport = serial
 lib_deps =
     ../
+    
+[env:pico]
+platform = raspberrypi
+board = pico
+framework = arduino
```

```suggestion
framework = arduino
lib_ldf_mode = chain+
```

Also add it to [Readme Known Issues/Limitations](https://github.com/micro-ROS/micro_ros_platformio#known-issueslimitations)